schema/mysql/v8/visibility/versioned/v1.1/advanced_visibility.sql

```diff
@@ -0,0 +1,132 @@
+-- Triggers to fix close_time NULL value to default value
```

Comment:
Could you attach the diffs between this schema and v57 to make it easier to review?

Reply:
I split the commit in two, so you should be able to see the difference.